extensions/search-notion/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "search-notion",
+  "title": "Search Notion",
+  "description": "Raycast Command for Searching Notion",
+  "icon": "command-icon.png",
+  "author": "reckoning-dev",
+  "version": "1.0.0",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "commandsearchnotion",
+      "title": "Search Notion",
+      "subtitle": "",
+      "description": "Search Notion Pages",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "cookie",
+      "type": "password",
+      "required": true,
+      "title": "Notion Search Cookie",
+      "description": "Your Personal Notion Search Cookie",
+      "placeholder": "Enter your Personal Notion Search Cookie",
+      "link": "https://www.notion.so"
+    },
+    {
+      "name": "spaceID",
+      "type": "password",
+      "required": true,
+      "title": "Notion Search spaceId",
```

It's the ID of the workspace, right?!

```suggestion
      "title": "Workspace ID",
```